internal/server/server.go

```diff
@@ -75,12 +76,14 @@ func (s *Server) Run() error {
 		ErrorHandler: oapiErrorHandler,
 	}
 
+	authMiddleware := auth.CreateAuthMiddleware(s.cfg)
 	router := chi.NewRouter()
 	router.Use(
 		middleware.RequestID,
 		middleware.Logger,
 		middleware.Recoverer,
 		oapimiddleware.OapiRequestValidatorWithOptions(swagger, &oapiOpts),
+		authMiddleware.AuthHandler,
```

Comment:
Heads-up for #303 that'll split the middlewares for user- and device-facing APIs.

Reply:
Should the Auth middleware be earlier in the chain? E.g. should we even validate before authenticating?

Reply:
right, we should not validate before auth. I've moved the auth middleware higher.